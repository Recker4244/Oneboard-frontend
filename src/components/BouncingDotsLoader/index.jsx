/* eslint-disable react/self-closing-comp */
/* eslint-disable react/jsx-indent */
/* eslint-disable no-unused-vars */

import React from 'react';
import './BouncingDotsLoader.css';

function BouncingDotsLoader(props) {
  return (
    <div className="bouncing-loader">
        <div></div>
        <div></div>
        <div></div>
    </div>

  );
}

export default BouncingDotsLoader;
