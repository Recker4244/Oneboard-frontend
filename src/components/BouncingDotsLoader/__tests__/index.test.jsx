import React from 'react';
import { render } from '@testing-library/react';
import BouncingDotsLoader from '../index';

describe('BouncingDotsLoader', () => {
  it('should renders the component correctly', () => {
    const { container } = render(<BouncingDotsLoader />);
    expect(container.firstChild).toMatchSnapshot();
  });
});

