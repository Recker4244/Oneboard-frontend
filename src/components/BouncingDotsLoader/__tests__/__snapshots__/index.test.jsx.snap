// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BouncingDotsLoader should renders the component correctly 1`] = `
<div
  class="bouncing-loader"
>
  <div />
  <div />
  <div />
</div>
`;
